docs/deployment/index.mdx

```diff
@@ -6,7 +6,94 @@ For environments that don't support running container images, Topaz can be run a
 
 ## Deploy as a Sidecar
 
-WIP
+Deploying topaz as a sidecar next to your applications is the most efficient way to use the authorizer as it minimizez the authorization latency.
+
+In this section we can follow an example deployment on minikube based on a golang backend application and topaz authorizer as a sidecar.
+
+Start by preparing minikube, you can follow the installation instructions [here](https://minikube.sigs.k8s.io/docs/start/).
+
+Check if your minikube cluster is running:
+```
+minikube status
+
+minikube
+type: Control Plane
+host: Running
+kubelet: Running
+apiserver: Running
+kubeconfig: Configured
+```
+#### Preparing our backend application container.
+1. Clone [todo-go-v2](https://github.com/aserto-demo/todo-go-v2) repository 
+2. Add the following docker file to the root of your repository
```

Comment:
I'm not sure if we want to change the aserto-demo projects, but if we would change it I think it would make more sense to build the backend container in a CI there as a public image